mteb/tasks/Retrieval/fra/FQuADRetrieval.py

```diff
@@ -0,0 +1,78 @@
+from __future__ import annotations
+
+import datasets
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class FQuADRetrieval(AbsTaskRetrieval):
+    _EVAL_SPLITS = ["test_hasAns", "valid_hasAns"]
```

Comment:
You can use `test` and `validation` splits and filter out the non existing answers in the `dataset_transform()` function using `datasets.filter()`, see docs [here]( https://huggingface.co/docs/datasets/v2.19.0/en/package_reference/main_classes#datasets.Dataset.filter)